app/models/user.rb

```diff
@@ -1,8 +1,9 @@
 class User < ApplicationRecord
   has_many :ratings
   validates :email, uniqueness: true
-  before_save :format_email
+  before_validation :format_email
 
+private
```

Comment:
Unless we plan on calling `User#format_email` on an instance outside this class, I think it's better to set call back methods as private methods.